guides/common/modules/con_using-kernelcare-plug-in.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Using_KernelCare_Plug-in_{context}"]
+= Using KernelCare Plug-in
```

Comment:
Not a complete review, but does it need an article?
```suggestion
= Using the KernelCare Plug-in
```

Reply:
I got mixed results:

````
$ find . -iname "*plugin*" -o -iname "*plug-in*"
./guides/common/modules/proc_uninstalling-google-gce-plugin.adoc
./guides/common/modules/proc_uninstalling-amazon-ec2-plugin.adoc
./guides/common/modules/proc_installing-kernelcare-plug-in.adoc
./guides/common/modules/proc_installing-vmware-plugin.adoc
./guides/common/modules/proc_installing-google-gce-plugin.adoc
./guides/common/modules/proc_installing-shellhooks-plugin.adoc
./guides/common/modules/proc_uninstalling-microsoft-azure-plugin.adoc
./guides/common/modules/con_shellhooks-plugin.adoc
./guides/common/modules/proc_installing-amazon-ec2-plugin.adoc
./guides/common/modules/proc_salt-installing-salt-plugin.adoc
./guides/common/modules/proc_enabling-the-templatesync-plugin.adoc
./guides/common/modules/proc_uninstalling-the-foreman-templates-plugin.adoc
./guides/common/modules/con_using-kernelcare-plug-in.adoc
./guides/common/modules/proc_installing-openscap-plugin.adoc
./guides/common/modules/proc_installing-microsoft-azure-plugin.adoc
./guides/common/modules/proc_configuring-the-templatesync-plugin.adoc
./guides/common/modules/proc_installing-webhooks-plugin.adoc
````

Does someone else want to weigh in? Any strong opinions?

Reply:
Not a native speaker, but IMHO it's either `Using X` or `Using the X plugin`

Reply:
Renamed.